### PR TITLE
fix(ui): Refactor colors batch 2

### PR DESCRIPTION
**Pull request by robinrendle**
This refactors green/blue and all the gray colors I happened to miss. 

[Project status.](https://docs.google.com/spreadsheets/d/1AD6E1mdSzl4a6tyKxM9kZC4GaQQmDzimFE6DtxKY0DI/edit#gid=211488664)

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/theme.tsx [hunk 1]
```diff
@@ -178,7 +178,7 @@ const badge = {
     indicatorColor: colors.purple,
   },
   new: {
-    background: `linear-gradient(90deg, ${colors.green}, ${colors.greenDark})`,
+    background: `linear-gradient(90deg, ${colors.green}, ${colors.green500})`,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils/theme.tsx:181]: where does `green` fit on the scale?

**Review by billyvg — COMMENTED** [review]
LGTM so far, but would like to checkout the percy snaps

File: src/sentry/static/sentry/app/utils/theme.tsx (hunk 1 above)
Comment (robinrendle) [reply src/sentry/static/sentry/app/utils/theme.tsx:181]: `green` -> `green400`

Comment (robinrendle) [reply src/sentry/static/sentry/app/utils/theme.tsx:181]: Good catch though. I’ll go through the colors one more time in cased I goofed up any others.

**Review by billyvg — APPROVED** [review]

**Pull request merged**